### DISCUSSION TITLE
How to update the embeddings with new or updated documents

### DISCUSSION THREAD

**Discussion by christophberger** [nicobrauchtgit/obsidian-Smart2Brain#86] [Q&A]
Hi Smart2Brain Team,

How can I trigger a re-index for new or updated documents? 

Neither the UI, nor the option page, nor the command list has an option for updating the embeddings.

**Comment by Leo310** [comment]
Reindexing should happen automatically in the background if the content of a note changes or if you restart obsidian.

**Comment by christophberger** [comment]
> Reindexing should happen automatically in the background if the content of a note changes

Ok, then I probably did not wait long enough. 

Unfortunately, I cannot retest. Running models locally appears to overload my computer to the point where the GUI freezes, so I decided to pull the plug on my ollama/Smart 2nd Brain test.

**Reply by Leo310** [reply]
Thanks for your feedback. Unfortunately, the current local setup still has its limitations. Would you mind sharing your computer specs and the models you are using, so we can understand what works and what doesn't?

**Reply by christophberger** [reply]
Sure. I used [`gemma` 7B](https://ollama.com/library/gemma:7b) and [`nomic-embed-text`](https://ollama.com/library/nomic-embed-text:v1.5) for the embeddings. My computer is a MacBook Pro with an M2 Pro Chipset and 16GB RAM.
